ui/desktop/tests/e2e/fixtures/electronTest.js

```diff
@@ -109,6 +109,7 @@ exports.electronTest = test.extend({
         env: {
           NODE_ENV: 'test',
           BYPASS_APP_UPDATER: true,
+          ...process.env,
```

Comment:
whats the purpose of adding `...process.env`? Was this a bug or was it not needed before?